src/material/button/button.md

```diff
@@ -45,6 +45,20 @@ not.
 </button>
 ```
 
+### Interactive disabled buttons
+Native disabled `<button>` elements cannot receive focus and do not dispatch any events. This can
+be problematic in some cases, because it can prevent the app from the telling the user why the
```

Comment:
Nit: remove comma after "cases" and remove extra "the" in `from the telling the user`